app/controllers/v1/messages_controller.rb

```diff
@@ -0,0 +1,9 @@
+module V1
+  class MessagesController < ApplicationController
+    def index
+      message = Message.order(Arel.sql('RANDOM()')).first.message
+      # message = Message.all.pluck(:message)
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.